framework/global/globalmodule.cpp

```diff
@@ -40,25 +46,37 @@ std::string GlobalModule::moduleName() const
 
 void GlobalModule::registerExports()
 {
-    ioc()->registerExport<IGlobalConfiguration>(moduleName(), new GlobalConfiguration());
+    ioc()->registerExport<IGlobalConfiguration>(moduleName(), s_globalConf);
     ioc()->registerExport<IInteractive>(moduleName(), new Interactive());
 }
 
 void GlobalModule::onInit()
 {
+    //! --- Setup logger ---
+    using namespace haw::logger;
+    Logger* logger = Logger::instance();
+    logger->clearDests();
+
+    //! Console
+    logger->addDest(new ConsoleLogDest(LogLayout("${time} | ${type|5} | ${thread} | ${tag|10} | ${message}")));
+
+    //! File, this creates a file named "data/logs/MuseScore_yyMMdd.log"
+    std::string logsPath = io::path(s_globalConf->dataPath() + "/logs").toStdString();
```

Comment:
Maybe add a function to configuration:

io::path logsDirPath() { return dataPath() + "/logs" }

?

Reply:
added `logsPath` method to configuration